packages/app/src/components/Sidebar/Bookmarks.tsx

```diff
@@ -1,16 +1,120 @@
-import React, { FC } from 'react';
 
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { DevidedPagePath } from '@growi/core';
 import { useTranslation } from 'react-i18next';
+import { UncontrolledTooltip } from 'reactstrap';
+
+import { toastError } from '~/client/util/apiNotification';
+import { apiv3Get } from '~/client/util/apiv3-client';
+import { IPageHasId } from '~/interfaces/page';
+import { useCurrentUser, useIsGuestUser } from '~/stores/context';
+import loggerFactory from '~/utils/logger';
+
+const logger = loggerFactory('growi:BookmarkList');
+
+// TODO: Adjust pagination
```

Comment:
Pagination is not supported on Bookmarks sidebar

the comment should be written like the following.

```
// TODO:emove pagination and apply  scrolling (not infinity)
```